.github/workflows/config/auto-assign.yaml

```
@@ -0,0 +1,8 @@
+runOnDraft: false
```

actionlint (1.7.4)

unexpected key "runOnDraft" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)